### PR TITLE
Feature/move dummy data into initial state displaydata from redux store to instructors pages

### PR DESCRIPTION
**Pull request by Jie-chelchel**
What we did:
transferred the dummy data to the instructor reducer initial stat

What files were changed:
src/redux/reducers/instructorReducer

Why were these changed made:
to use the reducer initial state instead of the dummyData so when we have backend endpoints we can pull data correctly

coauthor @Rodgers31 

!(https://www.loom.com/share/f3d08ae3dc824825958f9c845806f731)

### CODE REVIEW

**Review by andrewskr90 — APPROVED** [review]
This is a nice feature, allowing the dummy data to be displayed now that redux is set up.

**Review by WillisLi — COMMENTED** [review]
Somethings of note: `approved` and `approved_by` have been moved from the `instructors` to the `instructor_list` table, which just shows if the instructor is qualified to teach that specific course. `instructor_bio` will be referred to as `bio` in the json. A `course` object in the `course_schedule` array will have a `session` property with a type array, holding all the `dates `and `times` of that specific scheduled course (e.g., `{ course_id: 3, size: 12, session: [{start_date, end_date, start_time, end_time}, {start_date, end_date, start_time, end_time}] }`).

File: src/components/pages/InstructorHome/InstructorCalender.js [hunk 1]
```diff
@@ -30,11 +28,12 @@ const InstructorCalender = ({ dummyData }) => {
 
   function dateCellRender(value) {
     const listData = getListData(value);
+
     return (
       <ul className="events">
         {listData.map(item => (
           <li key={item.content}>
-            <Badge status={item.type} text={item.description} />
+            <Badge status={item.type} text={item.course} />
```
Comment (fernando817mm) [inline src/components/pages/InstructorHome/InstructorCalender.js:36]: Line 36 had the prop "text={item.description}" before being changed to "item.course". Was this a typo or did BE change the schema and table causing this change? Other than that, looks good!

**Review by fernando817mm — APPROVED** [review]

File: src/components/pages/InstructorHome/InstructorCalender.js (hunk 1 above)
Comment (fernando817mm) [reply src/components/pages/InstructorHome/InstructorCalender.js:36]: Update: Took a look at a PR from earlier this evening and the BE modifying the schema appears to be the reason for the small change in names. Thank you!

Comment (Jie-chelchel) [reply src/components/pages/InstructorHome/InstructorCalender.js:36]: > Line 36 had the prop "text={item.description}" before being changed to "item.course". Was this a typo or did BE change the schema and table causing this change? Other than that, looks good!

I am not very clear about the two build-in function in this calendar. What I think happened here is : item.course is just the instructor.course_schedule.subject as you can see in line 11 that setCourse update the value of course with itemz.subject. I am still having a lot of questions about this calendar. Right now, for a given case like 8, there will be the same event on the 8th of every month. How to populate the calendar with a specific date is the question.   Even if we transfer the date into a string, and use the string as a case, how are we going to deal with all the different dates. We cannot hard code the case numbers. If anyone has a better understanding about the calendar, please share. Thanks.

**Comment by Jie-chelchel** [comment]
> Somethings of note: `approved` and `approved_by` have been moved from the `instructors` to the `instructor_list` table, which just shows if the instructor is qualified to teach that specific course. `instructor_bio` will be referred to as `bio` in the json. A `course` object in the `course_schedule` array will have a `session` property with a type array, holding all the `dates `and `times` of that specific scheduled course (e.g., `{ course_id: 3, size: 12, session: [{start_date, end_date, start_time, end_time}, {start_date, end_date, start_time, end_time}] }`).

Thanks for the note. We will definitely change the initial state later once we have a better idea about what we get from the api calls.

**Pull request merged**